models/src/main/kotlin/io/provenance/api/models/account/KeyManagementConfig.kt

```diff
@@ -1,7 +1,6 @@
 package io.provenance.api.models.account
 
 data class KeyManagementConfig(
-    val address: String,
-    val tokenPath: String,
     val plugin: String = "io.provenance.plugins.vault.VaultPlugin",
+    val pluginSpec: Any
```

Comment:
Only thing I don't love is this being an `Any` type

Reply:
fixed :D